Packs/CheckPointSandBlast/Integrations/CheckPointSandBlast/CheckPointSandBlast.py

```diff
@@ -0,0 +1,927 @@
+    features = argToList(args.get('features'))
+    image_ids = argToList(args.get('image_ids'))
+    image_revisions = [arg_to_number(image_revision)
+                       for image_revision in argToList(args.get('image_revisions'))]
+    reports = argToList(args.get('reports'))
+    method = args.get('method', '')
+    extracted_parts = argToList(args.get('extracted_parts'))
+
+    file_entry = demisto.getFilePath(file_id)
+
+    if not file_name:
+        file_name = file_entry['name']
+
+    file_type = os.path.splitext(file_name)[1]
+
+    if file_type != os.path.splitext(file_entry['name'])[1]:
+        raise ValueError('New file name must have the same extension as the original file!')
+
+    features = [FEATURE_BY_NAME[feature] for feature in features]
+
+    if 'all' in features:
+        features = ['te', 'av', 'extraction']
+
+    if len(image_ids) != len(image_revisions):
+        raise ValueError('Image IDs and image revisions must be of same length!')
+
+    if 'te' in features and {'pdf', 'summary'}.issubset(reports):
+        raise ValueError(
+            'Requesting for PDF and summary reports simultaneously is not supported!'
+        )
+
+    if method != 'clean':
+        extracted_parts_codes = None
+    else:
+        extracted_parts_codes = [
+            EXTRACTED_PARTS_CODE_BY_DESCRIPTION[extracted_part]
+            for extracted_part in extracted_parts
+        ]
+
+    raw_output = client.upload_request(
+        file_path=file_entry['path'],
+        file_name=file_name,
+        file_type=file_type,
+        features=features,
+        image_ids=image_ids,
+        image_revisions=image_revisions,
+        reports=reports,
+        method=method,
+        extracted_parts_codes=extracted_parts_codes,
+    )
+
+    output = raw_output.get('response', {'': ''})
+    readable_output = get_analysis_readable_output(features, output, 'Upload')
+    output = get_analysis_context_output(output)
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='SandBlast.Upload',
+        outputs_key_field=['MD5', 'SHA1', 'SHA256'],
+        outputs=output,
+        raw_response=raw_output,
+    )
+
+
+def download_command(client: Client, args: Dict[str, Any]) -> Any:
+    """
+    Download a file from the server.
+    The command will be a bridge between the client request and the presented value to the user.
+    Get arguments inputted by the user and send them to the client.
+    Once a response has been received, process and return it so it can be sent to the user.
+
+    Args:
+        client (Client): Connection to the client class from which we can run the desired request.
+        args (Dict[str, Any]): Arguments passed down by the CLI to provide in the HTTP request.
+
+    Returns:
+        Any: File from the server.
+    """
+    file_id = args['file_id']
+    output = client.download_request(file_id)
+
+    return fileResult(filename=file_id, data=output)
```

Comment:
since this returns the file of the report, we should be returning InfoFile. Please update yml context outputs accordingly

Reply:
Alright, updated from File to InfoFile both command and contextpath